### PR TITLE
Update contributors

### PR DESCRIPTION
**Pull request by casperklein**
# Description

This combines two approaches to list all contributors.

1. All repository contributors are automatically added / updated with a daily workflow
2. Custom contributors can be added via the `@all-contributors` bot

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement (non-breaking change that does improve existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (README.md or the documentation under `docs/`)
- [ ] If necessary I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by georglauterbach — DISMISSED** [review]
LGTM 👍

**Review by polarathene — DISMISSED** [review]
This is going to remove the current contribution types? Were those going to be added back or automated in some way? Or are we no longer seeing any value in them?

**Comment by casperklein** [comment]
I didn't considered this as very important. That's the drawback in favor of the automated listing of all code contributors.

We still can add everyone to the "second list" with the all-contributors bot. So this possibility is still given if we want that.

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -1,24 +1,1647 @@
...
+        <a href=https://github.com/vivacarvajalito>
+            <img src=https://avatars.githubusercontent.com/u/1446654?v=4 width="100;"  style="border-radius:50%;align-items:center;justify-content:center;overflow:hidden;padding-top:10px" alt=vivacarvajalito/>
+            <br />
+            <sub style="font-size:14px"><b>vivacarvajalito</b></sub>
+        </a>
+    </td>
+</tr>
+</table>
+
+## Further Contributors
```
Comment (wernerfred) [inline CONTRIBUTORS.md:1636]: We should include an information why this section is separated from the first one. What does this contributors do different?

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -29,9 +1652,3 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 <!-- ALL-CONTRIBUTORS-LIST:END -->
 
 This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
-
-____
-
-Note:
-
-We started using [all-contributors](https://github.com/all-contributors/all-contributors) in July 2021. We will add contributors with their future PRs or Issues. If you are [one of the 200+](https://github.com/docker-mailserver/docker-mailserver/graphs/contributors) that contributed to the project in the past and would like to see your name here too, please reach out!
```
Comment (wernerfred) [inline CONTRIBUTORS.md:37]: Maybe we should keep this sentence for this section anyways

**Review by wernerfred — COMMENTED** [review]
Can we somehow make sure that someone that gets added to "further contributors" is not already on the "normal" contributors.

I'm still not sure on this whole topic as the main benefit (showing on which topic someone contributed) gets lost / in the background (bc there are 200+ normal contributors in front). 
We could rely on the github contributors list alone and have no effort at all.

File: CONTRIBUTORS.md (hunk 2 above)
Comment (casperklein) [reply CONTRIBUTORS.md]: Good point.

**Comment by polarathene** [comment]
I don't mind either way, as @wernerfred mentioned, we have [official github repo stats on contributors](https://github.com/docker-mailserver/docker-mailserver/graphs/contributors) which looks to be sorted by number of commits from contributors. All of us rank high on that and are visible.

From what I understood, this automates the alternative contributors document that was setup, and still allows for making additions (such as different contribution badges) via the existing bot if desired. Sounds win/win to me :man_shrugging:

**Comment by casperklein** [comment]
@wernerfred Is the current solution for you acceptable?

I am going to merge this the next days, if there is no veto from someone.

**Comment by wernerfred** [comment]
Do it :rocket:

**Review by georglauterbach — APPROVED** [review]

**Pull request merged**